learn/experimental/vector_search.mdx

```diff
@@ -58,10 +58,20 @@ curl \
   }'
 ```
 
-`documentTemplate` is an optional field you can use to send only relevant data to the model. `documentTemplate` must be a [Liquid template](https://shopify.github.io/liquid/).
+`apiKey` is a mandatory field when using an OpenAI embedder. [Generate an API key](https://platform.openai.com/api-keys) from your OpenAI account. Use [tier 2 keys](https://platform.openai.com/docs/guides/rate-limits/usage-tiers?context=tier-two) for optimal performance.
+
+`documentTemplate` is an optional field you can use to customize the data you send to the embedder. It is highly recommended you configure a custom template for your documents.
+
+<Capsule intent="tip" title="`documentTemplate` usage">
+`documentTemplate` must be a [Liquid template](https://shopify.github.io/liquid/). Use `{{ doc.attribute }}` to access document field values. Meilisearch also exposes a `{{ fields }}` array containing one object per document field, which you may access with `{{ field.name }}` and `{{ field.value }}`.
```

Comment:
Thank you very much for the addition, but I feel we really should specify the behavior if a document doesn't contain an attribute referenced in the template.

If you need to shorten the text, then let's give up on the more niche `fields`.

```suggestion
`documentTemplate` must be a [Liquid template](https://shopify.github.io/liquid/). Use `{{ doc.attribute }}` to access the `attribute` field value of your documents. Any field you refer to in this way must exist in all documents or an error will be raised at indexing time.
```